tests/ImageSharp.Tests/Formats/Png/PngEncoderTests.cs

```diff
@@ -449,44 +449,15 @@ public void Encode_PreserveTrns(string imagePath, PngBitDepth pngBitDepth, PngCo
         TestFile testFile = TestFile.Create(imagePath);
         using Image<Rgba32> input = testFile.CreateRgba32Image();
         PngMetadata inMeta = input.Metadata.GetPngMetadata();
-        Assert.True(inMeta.HasTransparency);
+        Assert.True(inMeta.TransparentColor.HasValue);
 
         using MemoryStream memStream = new();
         input.Save(memStream, PngEncoder);
         memStream.Position = 0;
         using Image<Rgba32> output = Image.Load<Rgba32>(memStream);
         PngMetadata outMeta = output.Metadata.GetPngMetadata();
-        Assert.True(outMeta.HasTransparency);
-
-        switch (pngColorType)
```

Comment:
We should validate `metadata.PngBitDepth/PngColorType` against the expected vaules `pngBitDepth` & `pngColorType`.

Reply:
Done